invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/Location/LocationList/components/InternalLocationList/InternalLocationList.js

```diff
@@ -20,7 +26,7 @@ export default class InternalLocationList extends Component {
   }
 
   handleOnRefClick(itemPid) {
-    // TODO: EDITOR, implement edit form
+    // TODO: navigate to item edit form
```

Comment:
is this missing on purpose?

Reply:
yes I don't have the item edit form yet :)